doc/source/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,182 @@
+                set number              				= 20000
+                set density particles  	 				= 2500
+                set young modulus particles         	= 100000000
+                set poisson ratio particles          	= 0.24
+                set restitution coefficient particles	= 0.97
+                set friction coefficient particles      = 0.3
+                set rolling friction particles         	= 0.01
+        end
+        set young modulus wall            				= 100000000
+        set poisson ratio wall            				= 0.24
+        set restitution coefficient wall           		= 0.85
+        set friction coefficient wall         			= 0.35
+        set rolling friction wall         	      	  	= 0.01
+    end
+
+
+Model parameters
+~~~~~~~~~~~~~~~~~
+
+The load balancing method used is the dynamic method, more details are explained in the `Rotating Drum example <../rotating-drum/rotating-drum.html>`_. 
+
+.. code-block:: text
+
+    subsection model parameters
+      set contact detection method 		   	 = dynamic
+      set dynamic contact search size coefficient	 = 0.8
+      set neighborhood threshold			 = 1.3
+      set load balance method				 = dynamic
+      set load balance threshold			 = 0.5
+      set dynamic load balance check frequency		 = 100000
+      set particle particle contact force method         = hertz_mindlin_limit_overlap
+      set particle wall contact force method             = nonlinear
+      set integration method				 = velocity_verlet
+    end
+
+Boundary Condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The rotation of the cylinder is applied using a rotational boundary condition with a value of 8 rad/s over the x axis.
+
+.. code-block:: text
+
+    subsection DEM boundary conditions
+      set number of boundary conditions         = 1
+        subsection boundary condition 0
+            set boundary id					= 0
+            set type              				= rotational
+            set rotational speed				= 8
+            set rotational vector x				= 1
+            set rotational vector y				= 0
+            set rotational vector z				= 0
+        end
+    end
+
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The simulation runs for 12 seconds of real time.
+
+.. code-block:: text
+
+    subsection simulation control
+      set time step                 		 = 5e-6
+      set time end       			 = 12
+      set log frequency				 = 2000
+      set output frequency            		 = 2000
+      set output path                  	 	 = ./output_dem/
+    end
+
+Running the simulation
+----------------------
+The simulation is launched on two steps:
+ 
+For packing: 
+
+.. code-block:: text
+
+   mpirun -np 6 dem_3d packing_rotating_drum.prm
+
+For cylinder rotation: 
+
+.. code-block:: text
+ 
+   mpirun -np 6 dem_3d small_rotating_drum_dem.prm
+
+
+.. warning::
+	This example needs a simulation time of approximately 6 hours 6 cores.s.
+
+
+Results
+---------
+
+Velocity profile of particles during the simulation:
```

Comment:
Nice image. Would you like to explain a little bit more? Does it make sense? Why is the velocity profile like this?